Lib/multiprocessing/queues.py

```diff
@@ -319,6 +319,13 @@ def __init__(self, *, ctx):
         else:
             self._wlock = ctx.Lock()
 
+    def __del__(self):
```

Comment:
Hum, maybe I gone too far. This change is part on a larger plan to add ResourceWarning everywhere.